pages/index.tsx

```diff
@@ -81,9 +81,9 @@ export default function Home() {
                                         <IoNotificationsOutline size={48} />
                                     </div>
                                     <div className="w-5/6">
-                                        <h4 className="text-black items-center xl:text-2xl">
+                                        <h3 className="text-black items-center xl:text-2xl">
                                             Guaranteed notification
-                                        </h4>
```

Comment:
Can we also vertical align the text? It will look better on mobile

Reply:
fix in https://github.com/nubpro/nakvaksin/pull/14/commits/c49ef92867cd7f3ee411ff6c0778d46cd26ba1c5

Reply:
Eh vertical align, not horizontal align

Reply:
![image](https://user-images.githubusercontent.com/43839286/125152853-41bf8280-e182-11eb-829c-999cbfc75bb1.png)

this is not what you want kah :3

Reply:
fixed in https://github.com/nubpro/nakvaksin/pull/14/commits/9f4bc3e0c5dc00bc583bf1051b2ecc75a51409f4